ete4/gtdb_taxonomy/gtdbquery.py

```diff
@@ -60,9 +61,9 @@ def __init__(self, dbfile=None, taxdump_file=None, memory=False):
         if dbfile != DEFAULT_GTDBTAXADB and not os.path.exists(self.dbfile):
             print('GTDB database not present yet (first time used?)', file=sys.stderr)
             urlbase = ('https://github.com/etetoolkit/ete-data/raw/main'
-                       '/gtdb_taxonomy/gtdb202')
-            update_ete_data(f'{DEFAULT_GTDBTAXADB}.traverse.pkl', f'{urlbase}/gtdbtaxa.sqlite.traverse.pkl')
-            update_ete_data(f'{DEFAULT_GTDBTAXADUMP}', f'{urlbase}/gtdb202dump.tar.gz')
+                       '/gtdb_taxonomy/gtdblatest')
+            
+            update_ete_data(f'{DEFAULT_GTDBTAXADUMP}', f'{urlbase}/gtdbdump.tar.gz')
```

Comment:
I think we'll have to change this to `gtdb_latest_dump.tar.gz` (for reference, we can see the new name here https://github.com/etetoolkit/ete-data/tree/main/gtdb_taxonomy/gtdblatest)